tahoma_build.sh

```diff
@@ -5,6 +5,14 @@ export TOONZDIR=toonz/build/toonz/Release
 echo ">>> Copying stuff to $TOONZDIR/Tahoma.app/tahomastuff"
 cp -R stuff $TOONZDIR/Tahoma.app/tahomastuff
 
+echo ">>> Copying FFmpeg to $TOONZDIR/Tahoma.app/ffmpeg"
+# This assumes an FFmpeg directory is in Downloads & that the executables are directly in that directory
+# (not in ffmpeg/bin or etc.) - path may need to be adjusted
+cp -R ~/Downloads/ffmpeg $TOONZDIR/Tahoma.app/ffmpeg
```

Comment:
Copying ffmpeg like this might work for local builds, but will that still allow it to work on other machines that don't have ffmpeg installed?  My concern here is that all the libs that the ffmpeg executables need may not be included in the build by doing it this way and if a user doesn't have those, it will cause Tahoma to crash when it tries to use it.

I'm wondering if the executables for ffmpeg should actually be included in the macdeployqt command list, so it pulls the ffmpeg executables + any supporting libraries it needs.

Reply:
I did test on another mac & everything was fine. I've always just given my students a directory with only the ffmpeg executables & they always seemed to work fine. We did have a good deal of render crashing on complex scenes but it happened with any render format, not just gif/mp4.

Reply:
Ok. I know I've tried doing something similar with other applications and had problems.  Since you've confirmed it work as is, I'm ok with it.

I need to set up the Linux package builds in my other PR to do the same thing, though I can't do much for the Canon stuff.

Can we move this tahoma_build.sh script to the `ci-scripts/osx` ?  Can also name it travis-buildpkg.sh, similar to the linux one I created for this and use in OTX for building.  We'd need to put logic around the canon piece to only include it if it is detected.

Reply:
Sorry for being away for a few days.  Our dishswasher broke and flooded the kitchen and into the basement ceiling.  I've been working on getting our kitchen back in order.

From what I understand, this script is only going to be used to make the release and beta builds.  I'm not sure this needs to be incorporated into the travis builds.  Please correct me if I'm wrong.  Since we can't include the Canon SDK in the repo, any build for release or beta needs to be compiled on a local machine.